test/qa_plugins_test.cpp

```diff
@@ -147,17 +147,14 @@ const boost::ut::suite BasicPluginNodesConnectionTests = [] {
     };
 
     "Graph"_test = [] {
-        fg::graph                 flow_graph;
+        fg::graph flow_graph;
 
-        auto                      node_source_load     = context().loader.instantiate(names::fixed_source, "double");
-        auto                     &node_source          = flow_graph.add_node(std::move(node_source_load));
+        auto     &node_source     = context().loader.instantiate_in_graph(flow_graph, names::fixed_source, "double");
+        auto     &node_multiply_1 = flow_graph.make_node<builtin_multiply<double>>(2.0);
```

Comment:
We should use only either the default constructor and/or the one accepting a `prarameter_map`...

Reply:
TBH, I prefer value-based constructors to `parameter_map` ones. For plugin instantiation, only the `parameter_map`-based one can work.

Reply:
OK, then just have the `parameter_map`-based one only and mimick the default through an empty/default map?!?

Reply:
Great! :+1: